2025-04-13T15:50:06.2105293Z Requested labels: ubuntu-latest
2025-04-13T15:50:06.2105577Z Job defined at: microsoft/markitdown/.github/workflows/tests.yml@refs/pull/1176/merge
2025-04-13T15:50:06.2105679Z Waiting for a runner to pick up this job...
2025-04-13T15:50:07.7878536Z Job is waiting for a hosted runner to come online.
2025-04-13T15:50:10.3444980Z Job is about to start running on the hosted runner: GitHub Actions 756 (hosted)
2025-04-13T15:50:12.0841059Z Current runner version: '2.323.0'
2025-04-13T15:50:12.0865687Z ##[group]Operating System
2025-04-13T15:50:12.0866449Z Ubuntu
2025-04-13T15:50:12.0867029Z 24.04.2
2025-04-13T15:50:12.0867537Z LTS
2025-04-13T15:50:12.0868284Z ##[endgroup]
2025-04-13T15:50:12.0868987Z ##[group]Runner Image
2025-04-13T15:50:12.0869585Z Image: ubuntu-24.04
2025-04-13T15:50:12.0870116Z Version: 20250406.1.0
2025-04-13T15:50:12.0871241Z Included Software: https://github.com/actions/runner-images/blob/ubuntu24/20250406.1/images/ubuntu/Ubuntu2404-Readme.md
2025-04-13T15:50:12.0872655Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu24%2F20250406.1
2025-04-13T15:50:12.0873616Z ##[endgroup]
2025-04-13T15:50:12.0874164Z ##[group]Runner Image Provisioner
2025-04-13T15:50:12.0874788Z 2.0.422.1
2025-04-13T15:50:12.0875328Z ##[endgroup]
2025-04-13T15:50:12.0876441Z ##[group]GITHUB_TOKEN Permissions
2025-04-13T15:50:12.0878633Z Contents: read
2025-04-13T15:50:12.0879303Z Metadata: read
2025-04-13T15:50:12.0880036Z Packages: read
2025-04-13T15:50:12.0880701Z ##[endgroup]
2025-04-13T15:50:12.0883749Z Secret source: None
2025-04-13T15:50:12.0884828Z Prepare workflow directory
2025-04-13T15:50:12.1186908Z Prepare all required actions
2025-04-13T15:50:12.1223713Z Getting action download info
2025-04-13T15:50:12.3026367Z ##[group]Download immutable action package 'actions/checkout@v4'
2025-04-13T15:50:12.3027264Z Version: 4.2.2
2025-04-13T15:50:12.3028288Z Digest: sha256:ccb2698953eaebd21c7bf6268a94f9c26518a7e38e27e0b83c1fe1ad049819b1
2025-04-13T15:50:12.3029389Z Source commit SHA: 11bd71901bbe5b1630ceea73d27597364c9af683
2025-04-13T15:50:12.3030057Z ##[endgroup]
2025-04-13T15:50:12.3823729Z ##[group]Download immutable action package 'actions/setup-python@v5'
2025-04-13T15:50:12.3824477Z Version: 5.5.0
2025-04-13T15:50:12.3825147Z Digest: sha256:1fac95d751afd58314aebd2ab01d4bf3b3109a441917bd2cac2a22898ec494f7
2025-04-13T15:50:12.3826034Z Source commit SHA: 8d9ed9ac5c53483de85588cdf95a591a75ab9f55
2025-04-13T15:50:12.3826675Z ##[endgroup]
2025-04-13T15:50:12.6760076Z Complete job name: tests
2025-04-13T15:50:12.7462752Z ##[group]Run actions/checkout@v4
2025-04-13T15:50:12.7463869Z with:
2025-04-13T15:50:12.7464594Z   repository: microsoft/markitdown
2025-04-13T15:50:12.7465680Z   token: ***
2025-04-13T15:50:12.7466120Z   ssh-strict: true
2025-04-13T15:50:12.7466552Z   ssh-user: git
2025-04-13T15:50:12.7466987Z   persist-credentials: true
2025-04-13T15:50:12.7467459Z   clean: true
2025-04-13T15:50:12.7467898Z   sparse-checkout-cone-mode: true
2025-04-13T15:50:12.7468781Z   fetch-depth: 1
2025-04-13T15:50:12.7469263Z   fetch-tags: false
2025-04-13T15:50:12.7469692Z   show-progress: true
2025-04-13T15:50:12.7470123Z   lfs: false
2025-04-13T15:50:12.7470519Z   submodules: false
2025-04-13T15:50:12.7470956Z   set-safe-directory: true
2025-04-13T15:50:12.7471794Z ##[endgroup]
2025-04-13T15:50:12.9081425Z Syncing repository: microsoft/markitdown
2025-04-13T15:50:12.9083154Z ##[group]Getting Git version info
2025-04-13T15:50:12.9083893Z Working directory is '/home/runner/work/markitdown/markitdown'
2025-04-13T15:50:12.9084972Z [command]/usr/bin/git version
2025-04-13T15:50:12.9139566Z git version 2.49.0
2025-04-13T15:50:12.9165884Z ##[endgroup]
2025-04-13T15:50:12.9178907Z Temporarily overriding HOME='/home/runner/work/_temp/83da1766-9846-413e-b4b5-7a894e7d302b' before making global git config changes
2025-04-13T15:50:12.9180298Z Adding repository directory to the temporary git global config as a safe directory
2025-04-13T15:50:12.9183661Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/markitdown/markitdown
2025-04-13T15:50:12.9217425Z Deleting the contents of '/home/runner/work/markitdown/markitdown'
2025-04-13T15:50:12.9221317Z ##[group]Initializing the repository
2025-04-13T15:50:12.9225193Z [command]/usr/bin/git init /home/runner/work/markitdown/markitdown
2025-04-13T15:50:12.9282004Z hint: Using 'master' as the name for the initial branch. This default branch name
2025-04-13T15:50:12.9283365Z hint: is subject to change. To configure the initial branch name to use in all
2025-04-13T15:50:12.9284573Z hint: of your new repositories, which will suppress this warning, call:
2025-04-13T15:50:12.9285261Z hint:
2025-04-13T15:50:12.9286089Z hint: 	git config --global init.defaultBranch <name>
2025-04-13T15:50:12.9286967Z hint:
2025-04-13T15:50:12.9287580Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2025-04-13T15:50:12.9289526Z hint: 'development'. The just-created branch can be renamed via this command:
2025-04-13T15:50:12.9290938Z hint:
2025-04-13T15:50:12.9291746Z hint: 	git branch -m <name>
2025-04-13T15:50:12.9293112Z Initialized empty Git repository in /home/runner/work/markitdown/markitdown/.git/
2025-04-13T15:50:12.9300738Z [command]/usr/bin/git remote add origin https://github.com/microsoft/markitdown
2025-04-13T15:50:12.9333998Z ##[endgroup]
2025-04-13T15:50:12.9335207Z ##[group]Disabling automatic garbage collection
2025-04-13T15:50:12.9339228Z [command]/usr/bin/git config --local gc.auto 0
2025-04-13T15:50:12.9367497Z ##[endgroup]
2025-04-13T15:50:12.9369134Z ##[group]Setting up auth
2025-04-13T15:50:12.9375422Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2025-04-13T15:50:12.9406081Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2025-04-13T15:50:12.9674579Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2025-04-13T15:50:12.9704702Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2025-04-13T15:50:12.9932771Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2025-04-13T15:50:12.9969112Z ##[endgroup]
2025-04-13T15:50:12.9970380Z ##[group]Fetching the repository
2025-04-13T15:50:12.9979308Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --no-recurse-submodules --depth=1 origin +cb0f2dc1f6c53aa8e877d0d2fb1c3c57f84d96cf:refs/remotes/pull/1176/merge
2025-04-13T15:50:13.3903083Z From https://github.com/microsoft/markitdown
2025-04-13T15:50:13.3904096Z  * [new ref]         cb0f2dc1f6c53aa8e877d0d2fb1c3c57f84d96cf -> pull/1176/merge
2025-04-13T15:50:13.3927302Z ##[endgroup]
2025-04-13T15:50:13.3928317Z ##[group]Determining the checkout info
2025-04-13T15:50:13.3929520Z ##[endgroup]
2025-04-13T15:50:13.3934616Z [command]/usr/bin/git sparse-checkout disable
2025-04-13T15:50:13.3974129Z [command]/usr/bin/git config --local --unset-all extensions.worktreeConfig
2025-04-13T15:50:13.4000401Z ##[group]Checking out the ref
2025-04-13T15:50:13.4004164Z [command]/usr/bin/git checkout --progress --force refs/remotes/pull/1176/merge
2025-04-13T15:50:13.4244658Z Note: switching to 'refs/remotes/pull/1176/merge'.
2025-04-13T15:50:13.4245188Z 
2025-04-13T15:50:13.4245633Z You are in 'detached HEAD' state. You can look around, make experimental
2025-04-13T15:50:13.4246539Z changes and commit them, and you can discard any commits you make in this
2025-04-13T15:50:13.4247439Z state without impacting any branches by switching back to a branch.
2025-04-13T15:50:13.4248148Z 
2025-04-13T15:50:13.4248572Z If you want to create a new branch to retain commits you create, you may
2025-04-13T15:50:13.4249668Z do so (now or later) by using -c with the switch command. Example:
2025-04-13T15:50:13.4250565Z 
2025-04-13T15:50:13.4250945Z   git switch -c <new-branch-name>
2025-04-13T15:50:13.4251585Z 
2025-04-13T15:50:13.4251932Z Or undo this operation with:
2025-04-13T15:50:13.4252510Z 
2025-04-13T15:50:13.4252816Z   git switch -
2025-04-13T15:50:13.4253225Z 
2025-04-13T15:50:13.4253915Z Turn off this advice by setting config variable advice.detachedHead to false
2025-04-13T15:50:13.4254971Z 
2025-04-13T15:50:13.4256116Z HEAD is now at cb0f2dc Merge 44a77509f36a5a369d08e81a72c796f0d29f940a into 3fcd48cdfc651cbf508071c8d2fb7d82aeb075de
2025-04-13T15:50:13.4259943Z ##[endgroup]
2025-04-13T15:50:13.4291082Z [command]/usr/bin/git log -1 --format=%H
2025-04-13T15:50:13.4312970Z cb0f2dc1f6c53aa8e877d0d2fb1c3c57f84d96cf
2025-04-13T15:50:13.4640867Z ##[group]Run actions/setup-python@v5
2025-04-13T15:50:13.4641444Z with:
2025-04-13T15:50:13.4641839Z   python-version: 3.10
3.11
3.12

2025-04-13T15:50:13.4642338Z   check-latest: false
2025-04-13T15:50:13.4642910Z   token: ***
2025-04-13T15:50:13.4643317Z   update-environment: true
2025-04-13T15:50:13.4643793Z   allow-prereleases: false
2025-04-13T15:50:13.4644253Z   freethreaded: false
2025-04-13T15:50:13.4644678Z ##[endgroup]
2025-04-13T15:50:13.6263280Z ##[group]Installed versions
2025-04-13T15:50:13.6323037Z Successfully set up CPython (3.10.16)
2025-04-13T15:50:13.6339502Z Successfully set up CPython (3.11.11)
2025-04-13T15:50:13.6349874Z Successfully set up CPython (3.12.9)
2025-04-13T15:50:13.6351139Z ##[endgroup]
2025-04-13T15:50:13.6471465Z ##[group]Run pipx install hatch
2025-04-13T15:50:13.6472049Z [36;1mpipx install hatch[0m
2025-04-13T15:50:13.6526118Z shell: /usr/bin/bash -e {0}
2025-04-13T15:50:13.6526614Z env:
2025-04-13T15:50:13.6527111Z   pythonLocation: /opt/hostedtoolcache/Python/3.12.9/x64
2025-04-13T15:50:13.6527912Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.12.9/x64/lib/pkgconfig
2025-04-13T15:50:13.6528986Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.9/x64
2025-04-13T15:50:13.6529703Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.9/x64
2025-04-13T15:50:13.6530412Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.9/x64
2025-04-13T15:50:13.6531624Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.12.9/x64/lib:/opt/hostedtoolcache/Python/3.11.11/x64/lib:/opt/hostedtoolcache/Python/3.10.16/x64/lib
2025-04-13T15:50:13.6532727Z ##[endgroup]
2025-04-13T15:50:13.7925946Z creating virtual environment...
2025-04-13T15:50:13.8846382Z installing hatch...
2025-04-13T15:50:19.4646779Z done! ✨ 🌟 ✨
2025-04-13T15:50:19.4647846Z   installed package hatch 1.14.1, installed using Python 3.12.3
2025-04-13T15:50:19.4649006Z   These apps are now globally available
2025-04-13T15:50:19.4649601Z     - hatch
2025-04-13T15:50:19.4856876Z ##[group]Run cd packages/markitdown; hatch test
2025-04-13T15:50:19.4857327Z [36;1mcd packages/markitdown; hatch test[0m
2025-04-13T15:50:19.4907721Z shell: /usr/bin/bash -e {0}
2025-04-13T15:50:19.4908176Z env:
2025-04-13T15:50:19.4908482Z   pythonLocation: /opt/hostedtoolcache/Python/3.12.9/x64
2025-04-13T15:50:19.4908955Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.12.9/x64/lib/pkgconfig
2025-04-13T15:50:19.4909392Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.9/x64
2025-04-13T15:50:19.4909804Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.9/x64
2025-04-13T15:50:19.4910227Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.9/x64
2025-04-13T15:50:19.4910951Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.12.9/x64/lib:/opt/hostedtoolcache/Python/3.11.11/x64/lib:/opt/hostedtoolcache/Python/3.10.16/x64/lib
2025-04-13T15:50:19.4911601Z ##[endgroup]
2025-04-13T15:50:19.7158383Z Creating environment: hatch-test.py3.12
2025-04-13T15:50:19.7639006Z Installing project in development mode
2025-04-13T15:50:21.6629100Z Checking dependencies
2025-04-13T15:50:21.7079273Z Syncing dependencies
2025-04-13T15:50:26.3902487Z ============================= test session starts ==============================
2025-04-13T15:50:26.3903450Z platform linux -- Python 3.12.3, pytest-8.3.5, pluggy-1.5.0
2025-04-13T15:50:26.3904295Z rootdir: /home/runner/work/markitdown/markitdown/packages/markitdown
2025-04-13T15:50:26.3905030Z configfile: pyproject.toml
2025-04-13T15:50:26.3905629Z plugins: rerunfailures-14.0, anyio-4.9.0, xdist-3.6.1, mock-3.14.0
2025-04-13T15:50:26.3906263Z collected 172 items
2025-04-13T15:50:26.3906520Z 
2025-04-13T15:50:28.1042086Z tests/test_cli_misc.py ..                                                [  1%]
2025-04-13T15:51:03.7243083Z tests/test_cli_vectors.py ......F...........F...........F.....ssssssssss [ 27%]
2025-04-13T15:51:05.6552949Z ss..                                                                     [ 30%]
2025-04-13T15:51:05.9769817Z tests/test_module_misc.py ......ss.ss                                    [ 36%]
2025-04-13T15:51:09.5460271Z tests/test_module_vectors.py ........................F..............F... [ 61%]
2025-04-13T15:51:16.0876239Z ...........F.....sssssssssssssss.........F..............F.........       [100%]
2025-04-13T15:51:16.0877723Z 
2025-04-13T15:51:16.0878150Z =================================== FAILURES ===================================
2025-04-13T15:51:16.0878901Z _____________________ test_output_to_stdout[test_vector6] ______________________
2025-04-13T15:51:16.0879320Z 
2025-04-13T15:51:16.0879654Z shared_tmp_dir = PosixPath('/tmp/pytest-of-runner/pytest-0/pytest_tmp0')
2025-04-13T15:51:16.0881468Z test_vector = FileTestVector(filename='test_mskanji.csv', mimetype='text/csv', charset='cp932', url=None, must_include=['名前,年齢,住所', '佐藤太郎,30,東京', '三木英子,25,大阪', '髙橋淳,35,名古屋'], must_not_include=[])
2025-04-13T15:51:16.0882536Z 
2025-04-13T15:51:16.0882840Z     @pytest.mark.parametrize("test_vector", CLI_TEST_VECTORS)
2025-04-13T15:51:16.0883674Z     def test_output_to_stdout(shared_tmp_dir, test_vector) -> None:
2025-04-13T15:51:16.0884361Z         """Test that the CLI outputs to stdout correctly."""
2025-04-13T15:51:16.0884854Z     
2025-04-13T15:51:16.0885203Z         result = subprocess.run(
2025-04-13T15:51:16.0885648Z             [
2025-04-13T15:51:16.0885995Z                 "python",
2025-04-13T15:51:16.0886385Z                 "-m",
2025-04-13T15:51:16.0886765Z                 "markitdown",
2025-04-13T15:51:16.0887276Z                 os.path.join(TEST_FILES_DIR, test_vector.filename),
2025-04-13T15:51:16.0887805Z             ],
2025-04-13T15:51:16.0888381Z             capture_output=True,
2025-04-13T15:51:16.0888825Z             text=True,
2025-04-13T15:51:16.0889190Z         )
2025-04-13T15:51:16.0889489Z     
2025-04-13T15:51:16.0890058Z         assert result.returncode == 0, f"CLI exited with error: {result.stderr}"
2025-04-13T15:51:16.0891758Z         for test_string in test_vector.must_include:
2025-04-13T15:51:16.0892666Z >           assert test_string in result.stdout
2025-04-13T15:51:16.0893974Z E           assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |\n'
2025-04-13T15:51:16.0896863Z E            +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |\n' = CompletedProcess(args=['python', '-m', 'markitdown', '/home/runner/work/markitdown/markitdown/packages/markitdown/test...ut may not work\n  warn("Couldn\'t find ffmpeg or avconv - defaulting to ffmpeg, but may not work", RuntimeWarning)\n').stdout
2025-04-13T15:51:16.0898970Z 
2025-04-13T15:51:16.0899230Z tests/test_cli_vectors.py:67: AssertionError
2025-04-13T15:51:16.0899888Z ______________________ test_output_to_file[test_vector6] _______________________
2025-04-13T15:51:16.0900383Z 
2025-04-13T15:51:16.0900727Z shared_tmp_dir = PosixPath('/tmp/pytest-of-runner/pytest-0/pytest_tmp0')
2025-04-13T15:51:16.0902479Z test_vector = FileTestVector(filename='test_mskanji.csv', mimetype='text/csv', charset='cp932', url=None, must_include=['名前,年齢,住所', '佐藤太郎,30,東京', '三木英子,25,大阪', '髙橋淳,35,名古屋'], must_not_include=[])
2025-04-13T15:51:16.0903481Z 
2025-04-13T15:51:16.0903718Z     @pytest.mark.parametrize("test_vector", CLI_TEST_VECTORS)
2025-04-13T15:51:16.0904184Z     def test_output_to_file(shared_tmp_dir, test_vector) -> None:
2025-04-13T15:51:16.0904582Z         """Test that the CLI outputs to a file correctly."""
2025-04-13T15:51:16.0904873Z     
2025-04-13T15:51:16.0905178Z         output_file = os.path.join(shared_tmp_dir, test_vector.filename + ".output")
2025-04-13T15:51:16.0905573Z         result = subprocess.run(
2025-04-13T15:51:16.0905810Z             [
2025-04-13T15:51:16.0906009Z                 "python",
2025-04-13T15:51:16.0906463Z                 "-m",
2025-04-13T15:51:16.0906672Z                 "markitdown",
2025-04-13T15:51:16.0906898Z                 "-o",
2025-04-13T15:51:16.0907116Z                 output_file,
2025-04-13T15:51:16.0907409Z                 os.path.join(TEST_FILES_DIR, test_vector.filename),
2025-04-13T15:51:16.0907712Z             ],
2025-04-13T15:51:16.0907914Z             capture_output=True,
2025-04-13T15:51:16.0908397Z             text=True,
2025-04-13T15:51:16.0908595Z         )
2025-04-13T15:51:16.0908765Z     
2025-04-13T15:51:16.0909075Z         assert result.returncode == 0, f"CLI exited with error: {result.stderr}"
2025-04-13T15:51:16.0909563Z         assert os.path.exists(output_file), f"Output file not created: {output_file}"
2025-04-13T15:51:16.0909924Z     
2025-04-13T15:51:16.0910127Z         with open(output_file, "r") as f:
2025-04-13T15:51:16.0910404Z             output_data = f.read()
2025-04-13T15:51:16.0910700Z             for test_string in test_vector.must_include:
2025-04-13T15:51:16.0911030Z >               assert test_string in output_data
2025-04-13T15:51:16.0911700Z E               AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0912093Z 
2025-04-13T15:51:16.0912222Z tests/test_cli_vectors.py:96: AssertionError
2025-04-13T15:51:16.0912609Z ______________ test_input_from_stdin_without_hints[test_vector6] _______________
2025-04-13T15:51:16.0912880Z 
2025-04-13T15:51:16.0913081Z shared_tmp_dir = PosixPath('/tmp/pytest-of-runner/pytest-0/pytest_tmp0')
2025-04-13T15:51:16.0914034Z test_vector = FileTestVector(filename='test_mskanji.csv', mimetype='text/csv', charset='cp932', url=None, must_include=['名前,年齢,住所', '佐藤太郎,30,東京', '三木英子,25,大阪', '髙橋淳,35,名古屋'], must_not_include=[])
2025-04-13T15:51:16.0914587Z 
2025-04-13T15:51:16.0914762Z     @pytest.mark.parametrize("test_vector", CLI_TEST_VECTORS)
2025-04-13T15:51:16.0915210Z     def test_input_from_stdin_without_hints(shared_tmp_dir, test_vector) -> None:
2025-04-13T15:51:16.0915638Z         """Test that the CLI readds from stdin correctly."""
2025-04-13T15:51:16.0915923Z     
2025-04-13T15:51:16.0916278Z         test_input = b""
2025-04-13T15:51:16.0916632Z         with open(os.path.join(TEST_FILES_DIR, test_vector.filename), "rb") as stream:
2025-04-13T15:51:16.0917032Z             test_input = stream.read()
2025-04-13T15:51:16.0917334Z     
2025-04-13T15:51:16.0917525Z         result = subprocess.run(
2025-04-13T15:51:16.0917752Z             [
2025-04-13T15:51:16.0918162Z                 "python",
2025-04-13T15:51:16.0918410Z                 "-m",
2025-04-13T15:51:16.0918634Z                 "markitdown",
2025-04-13T15:51:16.0918927Z                 os.path.join(TEST_FILES_DIR, test_vector.filename),
2025-04-13T15:51:16.0919228Z             ],
2025-04-13T15:51:16.0919434Z             input=test_input,
2025-04-13T15:51:16.0919678Z             capture_output=True,
2025-04-13T15:51:16.0919917Z             text=False,
2025-04-13T15:51:16.0920118Z         )
2025-04-13T15:51:16.0920298Z     
2025-04-13T15:51:16.0920581Z         stdout = result.stdout.decode(locale.getpreferredencoding())
2025-04-13T15:51:16.0920931Z         assert (
2025-04-13T15:51:16.0921143Z             result.returncode == 0
2025-04-13T15:51:16.0921459Z         ), f"CLI exited with error: {result.stderr.decode('utf-8')}"
2025-04-13T15:51:16.0921808Z         for test_string in test_vector.must_include:
2025-04-13T15:51:16.0922106Z >           assert test_string in stdout
2025-04-13T15:51:16.0922731Z E           AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |\n'
2025-04-13T15:51:16.0923092Z 
2025-04-13T15:51:16.0923222Z tests/test_cli_vectors.py:129: AssertionError
2025-04-13T15:51:16.0923586Z _______________________ test_convert_local[test_vector9] _______________________
2025-04-13T15:51:16.0923837Z 
2025-04-13T15:51:16.0924654Z test_vector = FileTestVector(filename='test_mskanji.csv', mimetype='text/csv', charset='cp932', url=None, must_include=['名前,年齢,住所', '佐藤太郎,30,東京', '三木英子,25,大阪', '髙橋淳,35,名古屋'], must_not_include=[])
2025-04-13T15:51:16.0925214Z 
2025-04-13T15:51:16.0925449Z     @pytest.mark.parametrize("test_vector", GENERAL_TEST_VECTORS)
2025-04-13T15:51:16.0925813Z     def test_convert_local(test_vector):
2025-04-13T15:51:16.0926120Z         """Test the conversion of a local file."""
2025-04-13T15:51:16.0926417Z         markitdown = MarkItDown()
2025-04-13T15:51:16.0926649Z     
2025-04-13T15:51:16.0926859Z         result = markitdown.convert(
2025-04-13T15:51:16.0927241Z             os.path.join(TEST_FILES_DIR, test_vector.filename), url=test_vector.url
2025-04-13T15:51:16.0927592Z         )
2025-04-13T15:51:16.0927820Z         for string in test_vector.must_include:
2025-04-13T15:51:16.0928323Z >           assert string in result.markdown
2025-04-13T15:51:16.0928953Z E           AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0929888Z E            +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |' = <markitdown._base_converter.DocumentConverterResult object at 0x7f31f441aae0>.markdown
2025-04-13T15:51:16.0930374Z 
2025-04-13T15:51:16.0930521Z tests/test_module_vectors.py:69: AssertionError
2025-04-13T15:51:16.0930931Z _________________ test_convert_stream_with_hints[test_vector9] _________________
2025-04-13T15:51:16.0931205Z 
2025-04-13T15:51:16.0931885Z test_vector = FileTestVector(filename='test_mskanji.csv', mimetype='text/csv', charset='cp932', url=None, must_include=['名前,年齢,住所', '佐藤太郎,30,東京', '三木英子,25,大阪', '髙橋淳,35,名古屋'], must_not_include=[])
2025-04-13T15:51:16.0932443Z 
2025-04-13T15:51:16.0932645Z     @pytest.mark.parametrize("test_vector", GENERAL_TEST_VECTORS)
2025-04-13T15:51:16.0933045Z     def test_convert_stream_with_hints(test_vector):
2025-04-13T15:51:16.0933438Z         """Test the conversion of a stream with full stream info."""
2025-04-13T15:51:16.0933779Z         markitdown = MarkItDown()
2025-04-13T15:51:16.0934140Z     
2025-04-13T15:51:16.0934348Z         stream_info = StreamInfo(
2025-04-13T15:51:16.0934679Z             extension=os.path.splitext(test_vector.filename)[1],
2025-04-13T15:51:16.0935027Z             mimetype=test_vector.mimetype,
2025-04-13T15:51:16.0935322Z             charset=test_vector.charset,
2025-04-13T15:51:16.0935569Z         )
2025-04-13T15:51:16.0935740Z     
2025-04-13T15:51:16.0936066Z         with open(os.path.join(TEST_FILES_DIR, test_vector.filename), "rb") as stream:
2025-04-13T15:51:16.0936467Z             result = markitdown.convert(
2025-04-13T15:51:16.0936805Z                 stream, stream_info=stream_info, url=test_vector.url
2025-04-13T15:51:16.0937120Z             )
2025-04-13T15:51:16.0937355Z             for string in test_vector.must_include:
2025-04-13T15:51:16.0937680Z >               assert string in result.markdown
2025-04-13T15:51:16.0938528Z E               AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0939492Z E                +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |' = <markitdown._base_converter.DocumentConverterResult object at 0x7f31f63ba8d0>.markdown
2025-04-13T15:51:16.0939970Z 
2025-04-13T15:51:16.0940116Z tests/test_module_vectors.py:90: AssertionError
2025-04-13T15:51:16.0940529Z _______________ test_convert_stream_without_hints[test_vector9] ________________
2025-04-13T15:51:16.0940800Z 
2025-04-13T15:51:16.0941475Z test_vector = FileTestVector(filename='test_mskanji.csv', mimetype='text/csv', charset='cp932', url=None, must_include=['名前,年齢,住所', '佐藤太郎,30,東京', '三木英子,25,大阪', '髙橋淳,35,名古屋'], must_not_include=[])
2025-04-13T15:51:16.0942202Z 
2025-04-13T15:51:16.0942406Z     @pytest.mark.parametrize("test_vector", GENERAL_TEST_VECTORS)
2025-04-13T15:51:16.0942808Z     def test_convert_stream_without_hints(test_vector):
2025-04-13T15:51:16.0943200Z         """Test the conversion of a stream with no stream info."""
2025-04-13T15:51:16.0943531Z         markitdown = MarkItDown()
2025-04-13T15:51:16.0943764Z     
2025-04-13T15:51:16.0944093Z         with open(os.path.join(TEST_FILES_DIR, test_vector.filename), "rb") as stream:
2025-04-13T15:51:16.0944555Z             result = markitdown.convert(stream, url=test_vector.url)
2025-04-13T15:51:16.0944912Z             for string in test_vector.must_include:
2025-04-13T15:51:16.0945219Z >               assert string in result.markdown
2025-04-13T15:51:16.0945816Z E               AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0946707Z E                +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |' = <markitdown._base_converter.DocumentConverterResult object at 0x7f31f7bf4890>.markdown
2025-04-13T15:51:16.0947304Z 
2025-04-13T15:51:16.0947456Z tests/test_module_vectors.py:103: AssertionError
2025-04-13T15:51:16.0947847Z _____________________ test_convert_file_uri[test_vector9] ______________________
2025-04-13T15:51:16.0948331Z 
2025-04-13T15:51:16.0949038Z test_vector = FileTestVector(filename='test_mskanji.csv', mimetype='text/csv', charset='cp932', url=None, must_include=['名前,年齢,住所', '佐藤太郎,30,東京', '三木英子,25,大阪', '髙橋淳,35,名古屋'], must_not_include=[])
2025-04-13T15:51:16.0949588Z 
2025-04-13T15:51:16.0949788Z     @pytest.mark.parametrize("test_vector", GENERAL_TEST_VECTORS)
2025-04-13T15:51:16.0950156Z     def test_convert_file_uri(test_vector):
2025-04-13T15:51:16.0950465Z         """Test the conversion of a file:// URI."""
2025-04-13T15:51:16.0950763Z         markitdown = MarkItDown()
2025-04-13T15:51:16.0951007Z     
2025-04-13T15:51:16.0951222Z         result = markitdown.convert(
2025-04-13T15:51:16.0951603Z             Path(os.path.join(TEST_FILES_DIR, test_vector.filename)).as_uri(),
2025-04-13T15:51:16.0951971Z             url=test_vector.url,
2025-04-13T15:51:16.0952198Z         )
2025-04-13T15:51:16.0952430Z         for string in test_vector.must_include:
2025-04-13T15:51:16.0952735Z >           assert string in result.markdown
2025-04-13T15:51:16.0953320Z E           AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0954198Z E            +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |' = <markitdown._base_converter.DocumentConverterResult object at 0x7f31f9ef8260>.markdown
2025-04-13T15:51:16.0954700Z 
2025-04-13T15:51:16.0954850Z tests/test_module_vectors.py:139: AssertionError
2025-04-13T15:51:16.0955386Z _____________________ test_convert_data_uri[test_vector9] ______________________
2025-04-13T15:51:16.0955644Z 
2025-04-13T15:51:16.0956363Z test_vector = FileTestVector(filename='test_mskanji.csv', mimetype='text/csv', charset='cp932', url=None, must_include=['名前,年齢,住所', '佐藤太郎,30,東京', '三木英子,25,大阪', '髙橋淳,35,名古屋'], must_not_include=[])
2025-04-13T15:51:16.0956935Z 
2025-04-13T15:51:16.0957131Z     @pytest.mark.parametrize("test_vector", GENERAL_TEST_VECTORS)
2025-04-13T15:51:16.0957502Z     def test_convert_data_uri(test_vector):
2025-04-13T15:51:16.0957812Z         """Test the conversion of a data URI."""
2025-04-13T15:51:16.0958223Z         markitdown = MarkItDown()
2025-04-13T15:51:16.0958447Z     
2025-04-13T15:51:16.0958626Z         data = ""
2025-04-13T15:51:16.0958976Z         with open(os.path.join(TEST_FILES_DIR, test_vector.filename), "rb") as stream:
2025-04-13T15:51:16.0959428Z             data = base64.b64encode(stream.read()).decode("utf-8")
2025-04-13T15:51:16.0959766Z         mimetype = test_vector.mimetype
2025-04-13T15:51:16.0960084Z         data_uri = f"data:{mimetype};base64,{data}"
2025-04-13T15:51:16.0960350Z     
2025-04-13T15:51:16.0960563Z         result = markitdown.convert(
2025-04-13T15:51:16.0960825Z             data_uri,
2025-04-13T15:51:16.0961055Z             url=test_vector.url,
2025-04-13T15:51:16.0961280Z         )
2025-04-13T15:51:16.0961515Z         for string in test_vector.must_include:
2025-04-13T15:51:16.0961828Z >           assert string in result.markdown
2025-04-13T15:51:16.0962459Z E           AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0963430Z E            +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |' = <markitdown._base_converter.DocumentConverterResult object at 0x7f31f44e3020>.markdown
2025-04-13T15:51:16.0963943Z 
2025-04-13T15:51:16.0964106Z tests/test_module_vectors.py:160: AssertionError
2025-04-13T15:51:16.0964465Z =============================== warnings summary ===============================
2025-04-13T15:51:16.0965230Z ../../../../../.local/share/hatch/env/virtual/markitdown/H_uUSwZ6/hatch-test.py3.12/lib/python3.12/site-packages/pydub/utils.py:170
2025-04-13T15:51:16.0966474Z   /home/runner/.local/share/hatch/env/virtual/markitdown/H_uUSwZ6/hatch-test.py3.12/lib/python3.12/site-packages/pydub/utils.py:170: RuntimeWarning: Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work
2025-04-13T15:51:16.0967529Z     warn("Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work", RuntimeWarning)
2025-04-13T15:51:16.0967871Z 
2025-04-13T15:51:16.0968209Z -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
2025-04-13T15:51:16.0968653Z =========================== short test summary info ============================
2025-04-13T15:51:16.0969565Z FAILED tests/test_cli_vectors.py::test_output_to_stdout[test_vector6] - assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |\n'
2025-04-13T15:51:16.0971273Z  +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |\n' = CompletedProcess(args=['python', '-m', 'markitdown', '/home/runner/work/markitdown/markitdown/packages/markitdown/test...ut may not work\n  warn("Couldn\'t find ffmpeg or avconv - defaulting to ffmpeg, but may not work", RuntimeWarning)\n').stdout
2025-04-13T15:51:16.0972942Z FAILED tests/test_cli_vectors.py::test_output_to_file[test_vector6] - AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0974249Z FAILED tests/test_cli_vectors.py::test_input_from_stdin_without_hints[test_vector6] - AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |\n'
2025-04-13T15:51:16.0975701Z FAILED tests/test_module_vectors.py::test_convert_local[test_vector9] - AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0976843Z  +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |' = <markitdown._base_converter.DocumentConverterResult object at 0x7f31f441aae0>.markdown
2025-04-13T15:51:16.0978168Z FAILED tests/test_module_vectors.py::test_convert_stream_with_hints[test_vector9] - AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0979355Z  +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |' = <markitdown._base_converter.DocumentConverterResult object at 0x7f31f63ba8d0>.markdown
2025-04-13T15:51:16.0980741Z FAILED tests/test_module_vectors.py::test_convert_stream_without_hints[test_vector9] - AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0981914Z  +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |' = <markitdown._base_converter.DocumentConverterResult object at 0x7f31f7bf4890>.markdown
2025-04-13T15:51:16.0983089Z FAILED tests/test_module_vectors.py::test_convert_file_uri[test_vector9] - AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0984227Z  +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |' = <markitdown._base_converter.DocumentConverterResult object at 0x7f31f9ef8260>.markdown
2025-04-13T15:51:16.0985429Z FAILED tests/test_module_vectors.py::test_convert_data_uri[test_vector9] - AssertionError: assert '名前,年齢,住所' in '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |'
2025-04-13T15:51:16.0986542Z  +  where '| ���O | �N�� | �Z�� |\n| --- | --- | --- |\n| �������Y | 30 | ���� |\n| �O�؉p�q | 25 | ��� |\n| �����~ | 35 | ���É� |' = <markitdown._base_converter.DocumentConverterResult object at 0x7f31f44e3020>.markdown
2025-04-13T15:51:16.0987214Z ============ 8 failed, 133 passed, 31 skipped, 1 warning in 53.42s =============
2025-04-13T15:51:16.3925922Z ##[error]Process completed with exit code 1.
2025-04-13T15:51:16.4046137Z Post job cleanup.
2025-04-13T15:51:16.4967147Z [command]/usr/bin/git version
2025-04-13T15:51:16.5008809Z git version 2.49.0
2025-04-13T15:51:16.5051826Z Temporarily overriding HOME='/home/runner/work/_temp/ff9fa0c4-2fee-4735-b896-17969d4d5b4f' before making global git config changes
2025-04-13T15:51:16.5053546Z Adding repository directory to the temporary git global config as a safe directory
2025-04-13T15:51:16.5057633Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/markitdown/markitdown
2025-04-13T15:51:16.5092407Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2025-04-13T15:51:16.5125583Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2025-04-13T15:51:16.5359932Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2025-04-13T15:51:16.5381827Z http.https://github.com/.extraheader
2025-04-13T15:51:16.5394204Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2025-04-13T15:51:16.5424519Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2025-04-13T15:51:16.5760556Z Cleaning up orphan processes
